src/tests/integration/lib/adapterSetup.ts

```diff
@@ -79,6 +79,24 @@ export class AdapterSetup {
 		await copy(tarballPath, path.resolve(this.testDir, tarballName));
 		await unlink(tarballPath);
 
+		// Let npm remove the adapter in the package-lock.json file(s),
+		// so that the installation in the following step
+		// won't grab the cached files.
+		// See https://github.com/ioBroker/testing/issues/612
+		debug("Removing the adapter from package-lock.json");
+		await executeCommand(
+			"npm",
+			[
+				"uninstall",
+				this.adapterFullName,
+				"--package-lock-only",
+				"--omit=dev",
```

Comment:
The `npm i` command below (line 122?) still uses the now outdated `--production` instead of `--omit=dev`. Can you update that call too?

Reply:
No I think a unit test isn't necessary. The "unrelated" changes may be necessary because npm has removed the flag since the code was originally written.